### PR TITLE
Ks/update nodes cs cleanup

### PR DESCRIPTION
**Pull request by krehermann**
Cleanup and simplify code. Extract funcs that are needed in `chainlink-deployment`. Fix typos, comment and adding logging

### CODE REVIEW

File: deployment/keystone/changeset/update_node_capabilities.go [hunk 1]
```diff
@@ -39,40 +56,19 @@ func (req *UpdateNodeCapabilitiesRequest) Validate() error {
 	if !exists {
 		return fmt.Errorf("registry chain selector %d does not exist", req.RegistryChainSel)
 	}
-	return nil
-}
-
-type UpdateNodeCapabilitiesImplRequest struct {
```
Comment (krehermann) [inline deployment/keystone/changeset/update_node_capabilities.go:45]: rm dead code

File: deployment/keystone/changeset/update_node_capabilities.go [hunk 2]
```diff
@@ -17,15 +18,31 @@ var _ deployment.ChangeSet = UpdateNodeCapabilities
...
+		P2PKey:              p2p,
+		EncryptionPublicKey: enc,
+	}, nil
+}
+
+// UpdateNodeCapabilitiesRequest is a request to set the capabilities of nodes in the registry
+type UpdateNodeCapabilitiesRequest = MutateNodeCapabilitiesRequest
+
+// MutateNodeCapabilitiesRequest is a request to change the capabilities of nodes in the registry
+type MutateNodeCapabilitiesRequest struct {
```
Comment (krehermann) [inline deployment/keystone/changeset/update_node_capabilities.go:37]: both update and append use the same form. unify and leverage type aliases

File: deployment/keystone/types.go [hunk 3]
```diff
@@ -202,32 +211,41 @@ type DonCapabilities struct {
...
+				return nil, fmt.Errorf("failed to get admin address for node %s: %w", node.ID, err)
 			}
+			out[node.ID] = NodeOperator(dc.Name, a)
+
 		}
 	}
 	return out, nil
 }
 
+func NodeOperator(name string, adminAddress string) capabilities_registry.CapabilitiesRegistryNodeOperator {
```
Comment (krehermann) [inline deployment/keystone/types.go:228]: refactor common util into func.

File: deployment/keystone/state.go [hunk 4]
```diff
@@ -40,7 +41,7 @@ func (cs ContractSet) View() (view.KeystoneChainView, error) {
 	return out, nil
 }
 
-func GetContractSets(req *GetContractSetsRequest) (*GetContractSetsResponse, error) {
+func GetContractSets(lggr logger.Logger, req *GetContractSetsRequest) (*GetContractSetsResponse, error) {
```
Comment (krehermann) [inline deployment/keystone/state.go:44]: better logging

**Review by bolekk — APPROVED** [review]

**Review by vyzaldysanchez — APPROVED** [review]

**Pull request merged**